auditlog/models/log.py

```diff
@@ -49,6 +51,15 @@ def write(self, vals):
             vals.update({"model_name": model.name, "model_model": model.model})
         return super().write(vals)
 
+    def show_res_ids(self):
+        self.ensure_one()
+        return {
+            "type": "ir.actions.act_window",
+            "view_mode": "tree,form",
+            "res_model": self.model_id.model,
+            "domain": [("id", "in", safe_eval(self.res_ids))],
+        }
```

Comment:
Perhaps to increase coverage, this function can just be called once from a test.